qiskit_algorithms/optimizers/adam_amsgrad.py

```diff
@@ -198,20 +197,12 @@ def load_params(self, load_dir: str) -> None:
         t = t[1:-1]
         self._t = np.fromstring(t, dtype=int, sep=" ")
 
-    @deprecate_arg("objective_function", new_alias="fun", since="0.19.0")
-    @deprecate_arg("initial_point", new_alias="fun", since="0.19.0")
-    @deprecate_arg("gradient_function", new_alias="jac", since="0.19.0")
     def minimize(
         self,
         fun: Callable[[POINT], float],
         x0: POINT,
         jac: Callable[[POINT], POINT] | None = None,
         bounds: list[tuple[float, float]] | None = None,
-        # pylint:disable=unused-argument
-        objective_function: Callable[[np.ndarray], float] | None = None,
-        initial_point: np.ndarray | None = None,
-        gradient_function: Callable[[np.ndarray], float] | None = None,
-        # ) -> Tuple[np.ndarray, float, int]:
     ) -> OptimizerResult:  # TODO find proper way to deprecate return type
```

Comment:
```suggestion
    ) -> OptimizerResult:
```
We can get rid of this TODO - the optimize this replaced returned a Tuple as commented out - this changed that but whatever was intended is no longer needed.